src/main/java/com/rocketden/main/game_object/Player.java

```diff
@@ -18,12 +19,12 @@
     private PlayerCode playerCode;
 
     // Submissions in order of first-to-last submitted.
-    private List<Submission> submissions;
+    private List<Submission> submissions = new ArrayList<>();
```

Comment:
This is just setting the base value, correct? (And the other same changes as well?)

Reply:
Yeah I believe so, basically to make it default to an empty list rather than null